charts/kashti/Chart.yaml

```diff
@@ -1,4 +1,5 @@
 apiVersion: v1
 description: A Helm chart for Kubernetes
 name: kashti
-version: 0.0.1
+version: 0.3.0
```

Comment:
Jumping up to 0.3.0 as the chart version was a choice.

It seemed weird to me to revise the minor version number of the _chart_ to `0.1.0`, because I worried it would imply (to less experienced helmers) that it installs _app version_ `v0.1.0`. There's no reason that chart version and app version need to be in sync going forward (or ever, even), but this seemed like the right thing to do in this particular circumstance.

I'm willing to be challenged on this choice-- I'm not married to it.